src/themes/shared/slotRecipes/slider.ts

```diff
@@ -19,13 +19,13 @@ export const sliderSlotRecipe = defineSlotRecipe({
     track: {
       flex: '1',
       height: 'var(--slider-marker-track)',
-      bg: 'gray.300',
-      borderRadius: 'full',
+      bg: 'gray.200',
+      borderRadius: 'sm',
     },
     range: {
-      height: 'var(--slider-marker-inset)',
-      bg: 'gray.800',
-      borderRadius: 'full',
+      height: 'var(--slider-marker-track)',
```

Comment:
not 100% sure where the change is coming from but the new slider has a much smaller thumb. We had a ticket just recently to increase the size of it because it was difficult to use on mobile. Could we try to keep that? Thanks!
<img width="1296" height="260" alt="Screenshot 2025-12-16 at 16 29 32" src="https://github.com/user-attachments/assets/aafd4ab4-85bd-4507-a235-a4c7e9e24eeb" />